### ISSUE TITLE
Broken jQuery reference in header.php

### ISSUE THREAD

**Issue by arafalov** [roots/sage#126]
The jQuery reference line said:
    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.6.2/jquery.min.js"></script>

Which is obviously broken. adding "http:" fixes the issue. It is hard to notice because the next line has a local-script fallback which masks the issue.
